src/IntegrationTests/java/uk/gov/hmcts/reform/sscs/tya/NotificationsIt.java

```diff
@@ -484,7 +521,7 @@ public void shouldSendNotificationForSyaAppealCreatedRequestForAnOralHearing() t
 
         assertHttpStatus(response, HttpStatus.OK);
         verify(notificationClient, times(1)).sendEmail(any(), any(), any(), any());
-        verify(notificationClient, times(1)).sendSms(any(), any(), any(), any(), any());
+        verify(notificationClient, times(2)).sendSms(any(), any(), any(), any(), any());
```

Comment:
Why two notification for create appeal? I think we don't sent notification for rep while creating the appeal